Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/certificate/DccV1Parser.kt

```diff
@@ -13,18 +13,21 @@ import javax.inject.Inject
 
 @Reusable
 class DccV1Parser @Inject constructor(
-    @BaseGson private val gson: Gson
+    @BaseGson private val gson: Gson,
+    private val dccJsonSchemaValidator: DccJsonSchemaValidator,
 ) {
     fun parse(map: CBORObject, mode: Mode): Body = try {
-        map[keyHCert]?.run {
-            this[keyEuDgcV1]?.run {
-                val (rawBody, dcc) = this.toCertificate()
-                Body(
-                    raw = rawBody,
-                    parsed = dcc.toValidated(mode)
-                )
-            } ?: throw InvalidVaccinationCertificateException(ErrorCode.HC_CWT_NO_DGC)
-        } ?: throw InvalidVaccinationCertificateException(ErrorCode.HC_CWT_NO_HCERT)
+        val dgcCbor = map[keyHCert].let {
+            if (it == null) throw InvalidVaccinationCertificateException(ErrorCode.HC_CWT_NO_HCERT)
```

Comment:
It should be InvalidHealthCertificateException throughout this class. I thought I changed that, but it is still there

Reply:
Changed it. Good catch.